package.js

```diff
@@ -13,9 +18,9 @@ Package.onUse(function(api) {
   api.use('reactive-dict');
 
   api.use('meteorhacks:fast-render@2.3.2', ['client', 'server'], {weak: true});
+  api.use('cosmos:browserify', 'client');
```

Comment:
Let's the add the version here. 

~~~js
api.use('cosmos:browserify@0.1.3', 'client');
~~~